mava/systems/jax/builder.py

```diff
@@ -168,6 +183,14 @@ def trainer(
             System trainer.
         """
 
+        # Set the rng key for the trainer.
+        if len(self.store.trainer_keys) > 1:
```

Comment:
This check could then be moved based on above

Reply:
I updated the code :+1: